frontend/src/pages/videos/VideoAnalysis.tsx

```diff
@@ -28,11 +33,16 @@ function VideoAnalysis() {
         <Grid item xs={12}>
           <VideoCard video={video} showPlayer={false} />
         </Grid>
-        <Grid item xs={12} md={6}>
-          <Card>
-            <CriteriaBarChart video={video} />
-          </Card>
-        </Grid>
+        {shouldDisplayCharts && (
+          <Grid item xs={12}>
```

Comment:
I made the chart full width because I found it looked weird with only half the width of the player. But we can change it back if you prefer (`md={6}`).

Reply:
Looks good to me. We will reorganize the layout of this page as we add a few elements to it soon.

Fyi @adrausch

Reply:
Actually I reverted this change in 569d6c27f063863512e2a445648a3f7ee71ba781 since it's not related to the issue addressed here.